### DISCUSSION TITLE
v0.8.1 - Automation of Release

### DISCUSSION THREAD

**Discussion by U-C-S** [U-C-S/Hurl#87] [Announcements]
A minor release to test the release automation workflow.

## Application Updates
- Minor UI updates to Quick Add Rulesets, Edit URL
- Improved Error Handling
- Increase Font and Icon sizes
- Update `WPF-UI` to `v3.0-preview`
- BREAKING CHANGE: Rename `AutoRoutingRules` property to `Rulesets`

## Development Updates
- Release Automation through Github Actions. (yey 🥳)
- Updates to `UserSettings.schema.json`, now includes descriptions and input validations for few properties.

<hr /><em>This discussion was created from the release <a href='https://github.com/U-C-S/Hurl/releases/tag/v0.8.1'>v0.8.1 - Automation of Release</a>.</em>

**Comment by stamminator** [comment]
Happy to see the update! It's a small gripe, but in my opinion, a breaking change should constitute more than just a patch number bump. Semver says that should be a major version bump, but it you didn't want to crack the 1.0 egg just yet, I think 0.9 would've been a better choice.

**Reply by U-C-S** [reply]
True, I am aware and guilty of that 😅. I haven't noticed I made a breaking change until the moment I made a draft release while updating the release notes and poor commit messages and merging half-baked stuff into main I made in the past doesn't help much here. I want to start putting more effort into the release flow, branching experimental stuff as I got more time in my hands again.

**Comment by eidylon** [comment]
Now to get it added to the winget repository, so that we can do updates through winget, instead of manual installs!  😎

**Reply by U-C-S** [reply]
That could be a thing, I didn't think of that. But I am not sure if they accept unsigned applications.
